src/bridge/cockpitpackages.c

```diff
@@ -850,7 +851,9 @@ handle_packages (CockpitWebServer *server,
       goto out;
     }
 
-  bytes = cockpit_web_response_negotiation (filename, package->paths, &chosen, &error);
+  languages = cockpit_web_server_parse_languages (headers, NULL);
+
+  bytes = cockpit_web_response_negotiation (filename, package->paths, languages[0], &chosen, &error);
```

Comment:
Accept-Language headers can have multiple values, do we want to make an effort to fall back in the users preferred order?